code/modules/admin/verbs/SDQL2/SDQL_2.dm

```diff
@@ -198,7 +198,7 @@
 		state = SDQL2_STATE_ERROR;\
 		CRASH("SDQL2 fatal error");};
 
-ADMIN_VERB(debug, query_text, "Query Text", "", R_DEBUG, query_text as message)
+ADMIN_VERB(debug, sdql2_query, "SDQL2 Query", "Run an SDQL2 query.", R_DEBUG, query_text as message)
```

Comment:
```suggestion
ADMIN_VERB(debug, sdql2_query, "SDQL2 Query", "Run a SDQL2 Query", R_DEBUG, query_text as message)
```

Reply:
"a ess" versus "an ess"; the latter is correct

Reply:
I'm not really stickler for grammatical stuff, that was my only nitpick

Reply:
sick, we're good to go then